traits-stubs/README.rst

```diff
@@ -0,0 +1,53 @@
+=========================================
+traits-stubs: Type annotations for Traits
+=========================================
+
+This package contains stub files which are type annotations for the most
+commonly used classes in the Traits package.
+
+Type annotations are used by static type checkers
+to find common bugs without running the program. These merely recommend
+and does not enforce changes to the source code for api compatibility.
+
+These annotations have been tested with the `mypy` static type checker.
+
+Information on how to setup and run `mypy` can be found here:
+
+- Repository & Quickstart: https://github.com/python/mypy
+- Documentation: https://mypy.readthedocs.io/en/stable/index.html
+
+
+Installation
+------------
+- Checkout the `traits-stubs` repository.
+- Activate an environment with `traits` installed.
+- Install mypy by following the instructions found in the links above.
+- Run `pip install <location containing setup.py>` for traits-stubs.
+- Run `mypy` with `mypy <somefile.py>`.
+
+Note: `mypy` creates a `.mypy_cache` folder when run. This may be excluded
+from source control.
+
+Plugins
+------------
```

Comment:
reST nit: fix the underlines to match the title (else Sphinx and other reST processors will complain).